### PR TITLE
bump: containerd to 1.7.14

### PR DESCRIPTION
**Pull request by lilypan26**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: pkg/proto/nbcontract/v1/tlsbootstrappingconfig.proto [hunk 1]
```diff
@@ -1,12 +1,9 @@
 syntax = "proto3";
 package nbcontract.v1;
 import "pkg/proto/nbcontract/v1/featurestate.proto";
-
+ 
 message TLSBootstrappingConfig {
```
Comment (cameronmeissner) [inline pkg/proto/nbcontract/v1/tlsbootstrappingconfig.proto:5]: looks like you forgot to checkout master before creating your new branch :P

**Comment by cameronmeissner** [comment]
containerd changes lgtm I think, lets wait on abe2e's (if they would actually not flake...) and test builds first though

File: pkg/proto/nbcontract/v1/tlsbootstrappingconfig.proto (hunk 1 above)
Comment (lilypan26) [reply pkg/proto/nbcontract/v1/tlsbootstrappingconfig.proto]: oops good catch, fixed

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 2]
```diff
@@ -130,7 +130,7 @@ installStandaloneContainerd() {
     #if there is no containerd_version input from RP, use hardcoded version
     if [[ -z ${CONTAINERD_VERSION} ]]; then
         # pin 18.04 to 1.7.1
-        CONTAINERD_VERSION="1.7.7"
+        CONTAINERD_VERSION="1.7.14"
```
Comment (cameronmeissner) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:133]: lets also verify that we want to keep 1804 pinned to 1.7.1

File: schemas/manifest.cue [hunk 3]
```diff
@@ -58,7 +58,7 @@
         "pinned": {
             "1804": "1.7.1-1" // default in 1804 vhds.
         }
-        "edge": "1.7.7-1",  // edge is default in vhd.
+        "edge": "1.7.14-1",  // edge is default in vhd.
```
Comment (cameronmeissner) [inline schemas/manifest.cue:61]: for future reference, we only ever need to update this manifest.cue - `make generate` as well as the VHD build itself will use this file to `export` to manifest.json in cloud-init/artifacts

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh (hunk 2 above)
Comment (cameronmeissner) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:133]: confirmed offline that we'll keep 1804 where it is

File: schemas/manifest.cue (hunk 3 above)
Comment (lilypan26) [reply schemas/manifest.cue:61]: yep, anuj informed me a while back, sorry should've reported back

**Review by cameronmeissner — APPROVED** [review]
let's also make a note of updating the default abe2e VHDs to pick up the bump

**Pull request merged**